src/db.py

```diff
@@ -93,47 +99,53 @@ def add_user(self, user: dict):
             role="user",
             use_reminder=False
         )
-
         session.add(new_user)
         session.commit()
         session.refresh(new_user)
         id = new_user.id
         session.close()
-
         if id is not None:
+            _logger.warning("User has been added successfully. STATUS_OK")
             return STATUS_OK
         else:
+            _logger.warning("User has not been added. STATUS_FAIL")
             return STATUS_FAIL
 
+    def is_user_exist(self, user_id: int):
+        session = self._sm()
+        user = session.query(User).filter(User.tg_id == user_id).one()
+        if user.tg_id:
+            return True
+        else:
+            return False
+
     def add_receipt(self, receipt: dict):
         """
-
         :param receipt: {"user_id": int, "text": str}
         :return:
         """
         session = self._sm()
-
         user_id = session.query(User.id).filter(User.tg_id == receipt["tg_id"]).first()
         if user_id is None:
+            _logger.warning("User is not found. STATUS_RECEIPT_UNKNOWN_USER")
             return STATUS_RECEIPT_UNKNOWN_USER
         user_id = user_id[0]
-
         if session.query(Receipt.id).filter(Receipt.text == receipt["text"]).count() > 0:
+            _logger.warning("Receipt already exists in database. STATUS_RECEIPT_ALREADY_EXIST")
             return STATUS_RECEIPT_ALREADY_EXIST
-
         receipt = Receipt(**receipt)
         receipt.user_id = user_id
         receipt.status = "active"
         session.add(receipt)
         session.commit()
-
         session.refresh(receipt)
         id = receipt.id
         session.close()
-
         if id is not None:
+            _logger.warning("Receipt has been added successfully. STATUS_OK")
             return STATUS_OK
         else:
+            _logger.warning("Receipt has not been added. STATUS_FAIL")
```

Comment:
This this error. Let's add text representation to the message 

`_logger.warning("Receipt has not been added. STATUS_FAIL. %r", receipt)`